benchmarks/benchmark_averaging.py

```diff
@@ -57,11 +57,10 @@ def run_averager(index):
         dht = hivemind.DHT(initial_peers=initial_peers, start=True)
         initial_bits = bin(index % num_groups)[2:].rjust(nbits, "0")
         averager = hivemind.averaging.DecentralizedAverager(
-            peer_tensors[i],
+            peer_tensors[index],
```

Comment:
This fixes a bug in `master` (previously, all tensors were the same because all threads used the same `i` value from the closure).